main/models.py

```diff
@@ -33,7 +33,7 @@ def approved_comments(self):
 class Comment(models.Model):
     """Model for comments."""
 
-    post = models.ForeignKey('main.Post', related_name='comments')
+    post = models.ForeignKey('main.Post', related_name='comments', on_delete=models.CASCADE)
```

Comment:
or explained why it's necessary here/how it helps to achieve the goal of **current task**